docs/tutorials/tutorial_observables.ipynb

```diff
@@ -0,0 +1,1572 @@
+{
```

Comment:
<p>Maybe I just missed this somewhere in the text, but we should mention somewhere that the probabilities need to sum to one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Will add